packages/form-core/index.d.ts

```diff
@@ -0,0 +1,52 @@
+export { FocusMixin } from './src/FocusMixin.js';
+export { FormatMixin } from './src/FormatMixin.js';
```

Comment:
I thought that an index.d.ts is only needed if you reexport things from node_modules?

Reply:
Nope. Always needed. I triple checked 2 weeks ago

Reply:
was not needed after all... removed